exercises/practice/isbn-verifier/.meta/tests.toml

```diff
@@ -1,21 +1,28 @@
-# This is an auto-generated file. Regular comments will be removed when this
-# file is regenerated. Regenerating will not touch any manually added keys,
-# so comments can be added in a "comment" key.
+# This is an auto-generated file.
+#
+# Regenerating this file via `configlet sync` will:
+# - Recreate every `description` key/value pair
+# - Recreate every `reimplements` key/value pair, where they exist in problem-specifications
+# - Remove any `include = true` key/value pair (an omitted `include` key implies inclusion)
+# - Preserve any other key/value pair
+#
+# As user-added comments (using the # character) will be removed when this file
+# is regenerated, comments can be added via a `comment` key.
 
 [0caa3eac-d2e3-4c29-8df8-b188bc8c9292]
-description = "valid isbn number"
+description = "valid isbn"
 
 [19f76b53-7c24-45f8-87b8-4604d0ccd248]
 description = "invalid isbn check digit"
 
 [4164bfee-fb0a-4a1c-9f70-64c6a1903dcd]
-description = "valid isbn number with a check digit of 10"
+description = "valid isbn with a check digit of 10"
```

Comment:
This line should not be changed, as the change would make it say "international standard book number number".

Reply:
This change is the opposite, it's removing `number`.

Reply:
Thanks.  Yeah, I saw it removed in October by ee7.  Misread it here.